modules/core/src/views/orbit-view.js

```diff
@@ -79,28 +79,20 @@ class OrbitViewport extends Viewport {
 }
 
 export default class OrbitView extends View {
+  constructor(props) {
+    super(
+      Object.assign({}, props, {
+        type: OrbitViewport
```

Comment:
is this ViewportType?

Reply:
https://github.com/uber/deck.gl/blob/master/modules/core/src/views/view.js#L152